src/suggestions.js

```diff
@@ -57,8 +59,8 @@ export function getSuggestedQuery(element, variant) {
     return makeSuggestion('PlaceholderText', placeholderText, {variant})
   }
 
-  const textContent = normalize(element.textContent)
-  if (textContent) {
+  const textContent = normalize(getNodeText(element))
+  if (textContent && !element.matches(DEFAULT_IGNORE_TAGS)) {
```

Comment:
Sorry, I don't mean it should be the default ignore but rather whatever ignore configuration was passed to the query. Is that possible?

Reply:
Oh that's not necessary.  The only reason it would be suggesting getByText is if it got something lower down the list and afaict none of those would use ignore.  And the only 2 above byText are label and role neither of which have an ignore option

Reply:
And if you're already using text it won't suggest text again